hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmDirectoryInfo.java

```diff
@@ -208,20 +208,14 @@ public DirectoryInfo getProtobuf() {
     return pib.build();
   }
 
-  /**
-   * Parses DirectoryInfo protobuf and creates OmPrefixInfo.
-   * @param dirInfo
-   * @return instance of OmDirectoryInfo
-   */
-  public static OmDirectoryInfo getFromProtobuf(DirectoryInfo dirInfo) {
+  public static Builder newBuilderFromProtobufPartial(DirectoryInfo dirInfo) {
```

Comment:
Change it similarly as `OmBucketInfo`, i.e. use one method to include/exclude large fields.

Reply:
Ok, done.